bio/mehari/download-clinvar-db/wrapper.py

```
@@ -0,0 +1,34 @@
+__email__ = "till.hartmann@bih-charite.de"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+
+import re
+
+version_re = re.compile(r"(?P<date>\d{8})\+(?P<annonars>\d+\.\d+\.\d+)")
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+version = snakemake.params.get("version")
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)